### PR TITLE
Newpr555

### PR DESCRIPTION
**Pull request by zhang-hongjie**

### CODE REVIEW

File: dep/src/main/java/dep/badClassName.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+	}
+	
+	public void BadMethodName(){
+		System.out.println("asdfsa");
+		return;
+	}
+
+	public void BadMethodName2(){
+		System.out.println("asdfsa");
```
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: dep/src/main/java/dep/badClassName.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package dep;
+
+public class badClassName {
```
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00101)

File: dep/src/main/java/dep/badClassName.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)

File: dep/src/main/java/dep/badClassName.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+	}
+	
+	public void BadMethodName(){
```
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)

File: dep/src/main/java/dep/badClassName.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+		System.out.println("asdfsa");
```
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: dep/src/main/java/dep/badClassName.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+		return;
+	}
+
+	public void BadMethodName2(){
```
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)

**Comment by argus-hongjie** [comment]
SonarQube analysis reported 6 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 3 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: dep/src/main/java/dep/badClassName.java (hunk 1 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: dep/src/main/java/dep/badClassName.java (hunk 2 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00101)

File: dep/src/main/java/dep/badClassName.java (hunk 3 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)

File: dep/src/main/java/dep/badClassName.java (hunk 4 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)

File: dep/src/main/java/dep/badClassName.java (hunk 5 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: dep/src/main/java/dep/badClassName.java (hunk 6 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)

**Comment by argus-hongjie** [comment]
SonarQube analysis reported 75 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 72 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [pom.xml#L8](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L8): Tab characters found (this is the first occurrence) [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)
1. ![MINOR][MINOR] [pom.xml#L78](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L78): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L79](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L79): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L80](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L80): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L90](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L90): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L91](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L91): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L92](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L92): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L95](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L95): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L96](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L96): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L97](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/dep/pom.xml#L97): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: mysql51/pom.xml [hunk 7]
```diff
@@ -1,13 +1,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.zhj2074</groupId>
-  <artifactId>mysql51</artifactId>
+  <artifactId>mysql513</artifactId>
   <version>0.0.1-SNAPSHOT</version>
   
   <dependencies>
   		<dependency>
```
Comment (argus-hongjie) [inline mysql51/pom.xml]: MINOR Make this line start at column 5. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
MINOR Tab characters found (this is the first occurrence) [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)

**Comment by argus-hongjie** [comment]
SonarQube analysis reported 23 issues
* ![MINOR][MINOR] 23 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [pom.xml#L14](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L14): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L15](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L15): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L16](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L16): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L17](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L17): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L20](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L20): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L21](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L21): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L22](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L22): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L25](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L25): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L26](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L26): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L27](https://github.com/argus-hongjie/mysql/blob/c24f2e28a80e3d489395a05973b4cb6346cd092a/mysql51/pom.xml#L27): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: dep/src/main/java/dep/badClassName.java (hunk 1 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: dep/src/main/java/dep/badClassName.java (hunk 2 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00101)

File: dep/src/main/java/dep/badClassName.java (hunk 3 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)

File: dep/src/main/java/dep/badClassName.java (hunk 4 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)

File: dep/src/main/java/dep/badClassName.java (hunk 5 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)

File: dep/src/main/java/dep/badClassName.java (hunk 6 above)
Comment (argus-hongjie) [inline dep/src/main/java/dep/badClassName.java]: MINOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)

**Comment by argus-hongjie** [comment]
SonarQube analysis reported 75 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 72 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [pom.xml#L8](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L8): Tab characters found (this is the first occurrence) [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)
1. ![MINOR][MINOR] [pom.xml#L78](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L78): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L79](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L79): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L80](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L80): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L90](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L90): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L91](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L91): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L92](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L92): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L95](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L95): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L96](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L96): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L97](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/dep/pom.xml#L97): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: mysql51/pom.xml (hunk 7 above)
Comment (zhang-hongjie) [inline mysql51/pom.xml:8]: MINOR Make this line start at column 5. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
MINOR Tab characters found (this is the first occurrence) [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIllegalTabCheck)

**Comment by zhang-hongjie** [comment]
SonarQube analysis reported 23 issues
* ![MINOR][MINOR] 23 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [pom.xml#L14](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L14): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L15](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L15): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L16](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L16): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L17](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L17): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L20](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L20): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L21](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L21): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L22](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L22): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L25](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L25): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L26](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L26): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)
1. ![MINOR][MINOR] [pom.xml#L27](https://github.com/argus-hongjie/mysql/blob/9fae17008bde6e7e50773d2d3eaf11edb04f899e/mysql51/pom.xml#L27): Make this line start at column 7. [rule](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request closed without merging**